### ISSUE TITLE
Custom main and accent color.

### ISSUE THREAD

**Issue by GHXST01** [squidfunk/mkdocs-material#466]
Hey,

I absolutely love the theme and I'm using it for some documentation project I am doing whilst on my internship.
However, I would really like to change the colors of the theme to match the corporate colors, I have the hex of those, but I'm a bit stuck on how to update the .scss file.

What I tried was including an extra stylesheet as you mention in the docs, and this works great when I (for example) want to edit every individual component on it's own.

But things get tricky when I need the colors to work correctly on for example mobile view, the scss has some custom ruling for that, which I don't know how to convert to plain css.

I also tried including the scss file of the application-palette, and adding a new color to the array like so:
```
@each $name, $color in (
  "red":         $clr-red-400,
  ...
  "new-red": #E31836
)
```

But when I select the new-red as color in the YAML file, it's not updating, so clearly I'm doing something wrong here and I was hoping you could tell me what it is.

Thanks for the amazing theme :)

**Comment by squidfunk** [comment]
Please refer to the [customization guide](http://squidfunk.github.io/mkdocs-material/customization/#theme-development), it contains an example for changing the colors.

**Comment by GHXST01** [comment]
Can you replace "$clr-red-400;" with an hex code without facing issues?
Upon trying this the color does not update.

Op 17 sep. 2017 15:32 schreef "Martin Donath" <notifications@github.com>:

**Comment by squidfunk** [comment]
Yes, you can. Did you build the theme?

**Comment by GHXST01** [comment]
Actually doing what's suggested solved this issue.
http://squidfunk.github.io/mkdocs-material/customization/#theme-development

**Comment by chrissphinx** [comment]
Forgive my confusion here, but I've actually tried doing what is suggested and I don't see any changes reflected on localhost:8000. Any idea what I could be doing wrong? Is executing `yarn run build` required?

**Comment by ZacSweers** [comment]
Building a distributing a fork of mkdocs-material seems quite heavy-weight for setting a custom theme color. Would it not be possible to just support setting custom hex values in a palette?

**Comment by squidfunk** [comment]
You don’t need to recompile the theme. You can always just use extra CSS (see #874 for a discussion) to make your changes.

v5 (tracked in #1306) will make it even easier with CSS variables (implemented in #1409), which should be released in a few weeks.
